source/digits_hits/src/GateGlobalActorAdder.cc

```diff
@@ -0,0 +1,80 @@
+/**
+ *  @copyright Copyright 2019 The J-PET Gate Authors. All rights reserved.
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  @file GateGlobalActorAdder.cc
+ */
+
+#include "GateGlobalActorAdder.hh"
+#include "TMath.h"
+#include <iostream>
+
+GateGlobalActorAdder::GateGlobalActorAdder() {}
+
+GateGlobalActorAdder::~GateGlobalActorAdder() {}
+
+void GateGlobalActorAdder::processHit( const GateGlobalActorHit& hit )
```

Comment:
Here some comment how this function work might be useful.

Reply:
Done